Source/WebCore/PlatformGTK.cmake

```diff
@@ -120,7 +120,17 @@ if (ENABLE_BUBBLEWRAP_SANDBOX)
     list(APPEND WebCore_LIBRARIES Libseccomp::Libseccomp)
 endif ()
 
-if (ENABLE_SPEECH_SYNTHESIS)
```

Comment:
If SPEECH_SYNTHESIS is disabled but spiel is set to on, removing this check causes that the spiel bits get added to the build. Eihter a depend is added to spiel so it's disabled when SPEECH_SYNTHESIS gets disabled, or we keep this check here.